### DISCUSSION TITLE
세부 기획 마지막?

### DISCUSSION THREAD

**Discussion by Namhoon-Cho** [osamhack2022-v2/WEB_CLOUD_OmilZomil_NAVYeffect#33] [General]
전군 통계 페이지에서 보여줄 것들 어떤 컴포넌트 있을지? 그 외는 일단락된듯 합니다. IPScan을 reference삼아도 좋겠어요. 이 또한 오늘 내로 정해지겠지만,,, 기획 단계가 늦어져서 미안하게 생각하고, 수정 의견 있으면 회의 혹은 멘토링 시간에 기탄없이 말해주세요. 그 외의 시간에도 비번일 때 가능한 한 답변 드리도록 하겠습니다. 감사합니다.

**Comment by Jun0zo** [comment]
짝짝짝
